src/main/resources/templates/start.html

```diff
@@ -13,26 +13,14 @@
     <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
     <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
     <style>
-        .navbar-nav > li{
-            padding-left:70px;
-        }
-        h2 {
-            color: #2faf9f;
-        }
-        #two {
-            padding-top: 60px;
-            padding-left: 50px;
-            padding-right: 50px;
-            padding-down: 50px;
-        }
-        .Meal{
-            width: 150px;
-        }
-       .btn
-       {
-           width: 150px;
-       }
-    </style>
```

Comment:
It appears when I told InteliJ to fetch all remotes it missed some things. I did not intend to remove this, it can be readded. I only intended to remove the styling for the navbar as I moved it to the navbar fragment.